### DISCUSSION TITLE
How to combine side navigation component with react router?

### DISCUSSION THREAD

**Discussion by S-N-O-R-L-A-X** [UI5/webcomponents-react#4580] [Q&A] [answered]
Where can I use the Link Component in react router?

**Comment by Lukas742** [comment, marked as answer]
Hi @S-N-O-R-L-A-X 

you can use for example the headless [`useNavigate`](https://reactrouter.com/en/main/hooks/use-navigate) hook to achieve this.

I've created a [codeSandbox](https://codesandbox.io/p/sandbox/condescending-dubinsky-rjw79z?file=%2Fsrc%2FApp.tsx&selection=%5B%7B%22endColumn%22%3A1%2C%22endLineNumber%22%3A42%2C%22startColumn%22%3A1%2C%22startLineNumber%22%3A41%7D%5D) that shows a basic example of how the implementation could look like.

**Reply by S-N-O-R-L-A-X** [reply]
Thanks for your answer!
